### PR TITLE
Add Slack channels to project list on Projects page

### LINKED ISSUE #241: Add Slack channels to project list on Projects page
## Description
Our Projects page should include the Slack channel for projects, if they have one.
https://opensource.creativecommons.org/contributing-code/projects/

## Additional context
This will also entail updating the CC metadata YAML file in repos to include the Slack channel information and updating the data gathering script (https://github.com/creativecommons/ccos-scripts/tree/master/generate_projects_page) to use that data.

### PR DESCRIPTION
**Pull request by sp35**
## Fixes

Fixes #241  by @kgodey 

## Description

This adds links to the slack channels of some CC repositories.
[mark-slack.svg](https://github.com/sp35/creativecommons.github.io-source/blob/slackch/content/contributing-code/projects/mark-slack.svg) is a resized version of the SVG collected from [FontAwesome](https://fontawesome.com/), which is already credited in the readme.

## Technical details

This involves ~adding a key slack to repos.json file~ checking for slack channel name in the repos json and then creating a button in the projects HTML file.

## Screenshots

Screenshot from 2020-05-10 04-29-20

Screenshot from 2020-05-16 08-36-49

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by sp35** [comment]
Please do have look at the slack channels added.
Thank you!

File: databags/repos.json [hunk 1]
```diff
@@ -13,7 +13,8 @@
             },
             "name": ".github",
             "url": "https://github.com/creativecommons/.github",
-            "website": ""
+            "website": "",
+            "slack": ""
```
Comment (kgodey) [inline databags/repos.json:17]: @sp35 you cannot add this here, this is an auto-generated file. you'll need to add slack channel info to individual repos in the `.cc-metadata.yml` file e.g. https://github.com/creativecommons/vocabulary/blob/master/.cc-metadata.yml and then update this script to read that info if it exists. https://github.com/creativecommons/ccos-scripts/blob/master/generate_projects_page/generate_projects_page.py

**Comment by sp35** [comment]
@kgodey I have updated my commit to remove the changes made in repos.json.

File: databags/repos.json (hunk 1 above)
Comment (sp35) [reply databags/repos.json]: Oh my bad, this should have been obvious, can't believe I did I this!

**Comment by kgodey** [comment]
@sp35 I'll review soon! Can you also update the description of the CC metadata file here to include Slack? https://opensource.creativecommons.org/contributing-code/github-repo-guidelines/

**Comment by kgodey** [comment]
thanks @sp35! I've labeled this a "Great Contribution", thanks for your work across multiple repos on this. Feel free to make other PRs adding Slack channels to repos as well.

**Pull request merged**